tests/system/action/poll/test_start.py

```diff
@@ -1,23 +1,8 @@
+from openslides_backend.models.models import Poll
 from tests.system.action.base import BaseActionTestCase
 
 
 class PollStartActionTest(BaseActionTestCase):
```

Comment:
This class only holds `test_start_wrong_state` now. Can this test be integrated in the classes below?

Reply:
Integrated into one class below. I don't think, that it need to be in all classes.